docs/getting-started-with-sylius/custom-business-logic.rst

```diff
@@ -129,6 +129,89 @@ For 4 products:
 
 |
 
+Or if you are using unified API:
+
+First You need to pickup new cart:
+
+.. code-block:: bash
+
+    curl -X POST "https://master.demo.sylius.com/api/v2/shop/orders" -H "accept: application/ld+json"
+
+With empty body:
+
+.. code-block:: json
+
+    {}
+
+This should return a response with ``tokenValue`` which we would need for next API calls:
+
+.. code-block:: json
+
+    {
+        "shippingState": "string",
+        "tokenValue": "CART_TOKEN",
+        "id": 123,
+    }
+
+Then we need to add product to the cart but first it would be good to have any, You can use this call to retrieve some products:
```

Comment:
```suggestion
Then we need to add a product to the cart but first, it would be good to have any. You can use this call to retrieve some products:
```